test/unit/spec/v2/ion/transformer_spec.js

```diff
@@ -0,0 +1,207 @@
+import transform from 'v2/ion/transformer';
+import XHRFactorRequiredEmail from '../../../helpers/xhr/v2/FACTOR_REQUIRED_EMAIL';
+import XHRFactorVerificationRequiredPush from '../../../helpers/xhr/v2/FACTOR_VERIFICATION_REQUIRED_OKTA_PUSH';
+
+describe('v2/ion/transformer', function () {
+  it('returns result when invokes with invalid resp', () => {
+    expect(transform()).toBeNull();
```

Comment:
maybe a future item might be to return some specific error when we cant parse the response.